charters/ingestion-service/charter.md

```diff
@@ -0,0 +1,177 @@
+
+### Dependencies
+
+The Ingestion Service will be as loosely coupled as possible to schemas defined by the metadata team. The definition of a core domain model for data and metadata is therefore in scope for the Ingestion Service, as this core domain model must be shared between the Ingestion Service and Metadata Schema. This core domain model defines high level concepts (e.g. 'Biomaterial', 'Protocol', 'File'), and changes to this core model must be agreed between the responsible teams.
+
+The Ingestion Service depends on the Upload Service for ingestion of data, and requires notifications of new file upload events in order to link those files to the appropriate metadata and to reference the cloud locations of those files.
+
+The Ingestion Service has a dependency on the Data Store write APIs when storing data (writing bundles and files) upon completion of a submission. Additionally, the Ingestion Service has a dependency on the Data Store team to provide new bundle requirements, and the Metadata Schema team to provide the definition of bundle specifications, in order to support the generation of bundles from submitted data and metadata.
+
+All services and tools that are used by direct data contributors will be rigorously tested with users to ensure a high quality user experience, and will take advantage of user experience experts within the DCP team.
+
+### Scientific "guardrails"
+
+The Ingestion Service requires oversight on which data types are required within the DCP (for example, SmartSeq2 and 10x sequencing data, imaging data) and by extension which file formats are endorsed and acceptable within the DCP (e.g. FASTQ, BAM, TIFF).
+
+## Out-of-scope
+
+### Interfaces
+
+* Preview illustration of data in the user interface, for example as it "will appear" in the data portal
+* Dedicated views over the data in the submission interface, for example visualisation of all data for a particular project or from a particular tissue type
+* Searching over submitted data, beyond that required for retrieval of a given user's previous submissions
+
+### Core capabilities
+
+* Metadata specification definitions. Ingest will utilise metadata definitions defined by DCP-compliant JSON schemas, as defined by the metadata working group, and will ensure the Ingest API is schema-agnostic where possible, but will not actively define metadata specifications beyond the restrictions placed by the shared core domain model outlined above
+* Definition of bundle structures for the DCP datastore. Whilst the Ingest API will provide support for generating bundles in response to submissions, the Ingestion Service is not responsible for the specification of bundle structures - it is expected these will be provided either by the metadata team or to address specific consumer requirements
```

Comment:
I don't think bundle structure specifications are provided by the metadata team. I thought they were more constrained by the DS (to meet indexing requirements) and/or the Pipelines team (to meet analysis requirements).

Reply:
I'm going to resolve this, as this was discussed quite a lot amongst PMs and POs. The Data Store charter now says this:
> The Data Store team will own the process of collecting new bundle requirements from users and assessing them against the datastore design before sending them to the metadata team for the definition of a new spec. The Data Store owns the bundle use cases and bundle type definitions, while the precise specifications will be negotiated between the Metadata and other teams.

... and the Metadata Charter currently says this:
> Working with Ingestion Service to specify the entity structure for HCA metadata schema

It might be worth updating the Metadata Charter to reflect the definition of the bundle specification, which is really just a mapping of which entities go into which bundle type, as defined by the DSS. @lauraclarke do you have thoughts?

I'll add a suggestion to the Metadata Charter reflecting this comment

Reply:
This is done